pkg/command/root.go

```diff
@@ -107,7 +107,12 @@ func NewRootCmd() (*cobra.Command, error) {
 				rootCmd.RemoveCommand(matchedCmd)
 				rootCmd.AddCommand(cmd)
 			} else {
-				maskedPlugins = append(maskedPlugins, plugins[i].Name)
+				// As the `login` plugin is now part of the core Tanzu CLI command and not a plugin
+				// anymore, skip the `login` plugin from adding it to the markedPlugins array to avoid
```

Comment:
nit: maskedPlugins, and there is extra space in the comment in the next line.
otherwise lgtm